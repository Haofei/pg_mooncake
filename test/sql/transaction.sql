CREATE TABLE s (a int);
CREATE TABLE t (b int) USING columnstore;

BEGIN;
INSERT INTO s VALUES (1);
INSERT INTO t VALUES (2);
COMMIT;

SELECT * FROM s;
SELECT * FROM t;

DROP TABLE s, t;
