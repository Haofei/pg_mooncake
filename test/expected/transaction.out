CREATE TABLE s (a int);
CREATE TABLE t (b int) USING columnstore;
BEGIN;
INSERT INTO s VALUES (1);
INSERT INTO t VALUES (2);
COMMIT;
SELECT * FROM s;
 a 
---
 1
(1 row)

SELECT * FROM t;
 b 
---
 2
(1 row)

DROP TABLE s, t;
